base_classes/NXdata.nxdl.xml

```diff
@@ -115,6 +115,43 @@
 			the axes attribute can be omitted.
 		</doc>
 	</attribute>
+	<attribute name="default_slice">
+		<doc>
+			Which slice of data to show in a plot by default. This is useful especially for
+			datasets with more than 2 dimensions.
+
+			Should be an array of strings of length equal to the number of dimesions
+			in the data, with the following possible values:
+
+			* ".": All the data in this dimension should be included
+			* Integer: Only this slice should be used.
+			* String: Only this slice should be used. Use if AXISNAME_indices is a string
```

Comment:
I don't know why I have not noticed this before, but this should be `AXISNAME` not `AXISNAME_indices`.